package.json

```diff
@@ -28,14 +28,14 @@
     "node": ">=6.2.2"
   },
   "devDependencies": {
-    "@types/jest": "20",
+    "@types/jest": "^20.0.8",
     "@types/node": "7",
     "@types/qs": "6",
     "dts-bundle": "0",
     "gh-pages": "1",
     "glob": "7",
-    "jest": "23",
-    "ts-jest": "22",
+    "jest": "^23.6.0",
```

Comment:
typically we'd update these kinds of dependencies on a separate PR and make sure that nothing breaks in the process. What did you need from these updates?

Reply:
Nothing. It got updated automatically. I'll change it back.